src/models/subject.js

```
@@ -0,0 +1,19 @@
+const { Schema, model } = require('mongoose')
+
+const { SUBJECT } = require('~/consts/models')
+
+const subjectSchema = new Schema({
+  name: {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)